sdk/core/Azure.Core/generate_solution.ps1

```diff
@@ -4,7 +4,7 @@ $RepoRoot = Resolve-Path $PSScriptRoot\..\..\..
 $ObjDirectory = "$RepoRoot\artifacts\obj";
 dotnet restore $RepoRoot\eng\service.proj
 
-pushd $PSScriptRoot
+Push-Location $PSScriptRoot
```

Comment:
While for PS scripts avoiding aliases is recommended, @danieljurek we should probably not be spell-checking the code itself in scripts - doc comments, string literals, public APIs, etc. This seems like an overreach and probably a bit too verbose in most cases.